client/components/Profile.jsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Loader from "../components/Loader.jsx"
+
+const profileSubs = new SubsManager();
```

Comment:
Define the max number of subs and timeout for the subs, remembering the defaults is difficult and may not be the values you want?